python-checks/src/test/java/org/sonar/python/checks/GenericExceptionRaisedCheckTest.java

```diff
@@ -19,14 +19,26 @@
  */
 package org.sonar.python.checks;
 
+import java.util.Arrays;
 import org.junit.jupiter.api.Test;
 import org.sonar.python.checks.utils.PythonCheckVerifier;
 
 class GenericExceptionRaisedCheckTest {
 
   @Test
   void test() {
-    PythonCheckVerifier.verify("src/test/resources/checks/genericExceptionRaised.py", new GenericExceptionRaisedCheck());
+    PythonCheckVerifier.verify("src/test/resources/checks/genericException/genericExceptionRaised.py", new GenericExceptionRaisedCheck());
+  }
+
+  @Test
+  void test_multiple_files() {
+    PythonCheckVerifier.verify(
+      Arrays.asList(
```

Comment:
```suggestion
      List.of(
```

I think using `List.of` is more consistent with the codebase